### ISSUE TITLE
Training: maximizing the likelihood

### ISSUE THREAD

**Issue by MUCDK** [liyaguang/DCRNN#34]
Hi,

Thanks for your paper! I've got a question concerning the training process. You wrote that 'the entire network is trained by maximizing the likelihood of generating the target future time series using backpropagation through time'. Where can I find the likelihood part in the code? I can only see that gradients are calculated using tf.gradients and they're being applied using the Adam optimizer. 

Thank you very much!

**Comment by liyaguang** [comment]
Thanks for your interest in our work and sorry for the late reply. Here are some comments:

Training: Maximizing the likelihood
- Using MAE as the loss function can be seen as maximizing the likelihood with a certain type of Laplacian distribution. 
- Similarly, using MSE as the loss function can be seen as maximizing the likelihood with a Gaussian distribution with fixed variance.
To do relatively more general likelihood, you may refer to Variational inference (one example is VAE) for more information.
